spmv/run_spmv.jl

```diff
@@ -32,7 +32,19 @@ parsed_args = parse_args(ARGS, s)
 datasets = Dict(
     "oski" => [
         "Boeing/ct20stif",
-    ]
+        "Simon/olafu",
+        "Boeing/bcsstk35",
+        "Boeing/crystk02",
+        "Boeing/crystk03",
+        "Nasa/nasasrb",
+        "Rothberg/3dtube",
+        "Simon/raefsky4",
+        "Mulvey/finan512",
+        "Pothen/pwt",
+        "Cote/vibrobox",
+        "HB/saylr4",
+        "Gupta/gupta1"
+    ],
```

Comment:
looks like the missing matrices are named [`HB/lns_3937`](https://sparse.tamu.edu/HB/lns_3937) and [`HB/jpwh_991`](https://sparse.tamu.edu/HB/jpwh_991). Let's skip the dense 1000x1000 matrix.

Reply:
Got it - looks like both of these matrices are not symmetric though so I'm going to skip them.